ui/admin/app/templates/scopes/scope/auth-methods/auth-method/index.hbs

```diff
@@ -1,3 +1,4 @@
+<h2>{{@model.type}}</h2>
```

Comment:
Is this for development purposes?
<img width="582" alt="Screenshot 2021-11-10 at 08 36 02" src="https://user-images.githubusercontent.com/8390120/141123153-1faa867f-cb7f-4e75-961b-1949d0273928.png">

Reply:
That was definitely unintentional :sweat_smile: